scripts/release.sh

```diff
@@ -259,8 +273,8 @@ bump_version() {
   fi
 
   log "commiting and pushing the version bump to github"
-  git config --global user.email "$git_email"
-  git config --global user.name "$git_user"
+  git config --global user.email $git_email
+  git config --global user.name $git_user
```

Comment:
🤔 Not even the user needs the double quoting here¿? `git_user="Lacework Inc."`

Reply:
This was copied from the services account repo. Shall I raise a PR to update there also?

Reply:
😂 I don't even know what day is today. Please, disregard this comment.